assets/templates/mods.html.twig

```diff
@@ -0,0 +1,136 @@
+<titleInfo>
+  <title lang="eng">{{ elements.title }}</title>
+  {% if elements.subtitle is not empty%}
```

Comment:
Just to get a sense of how to talk about this - this template here includes many more fields ("elements") than we currently have the ability to populate with a view. Some will be available to us if we create (e.g) a publisher filter that only shows the publisher. Others might need fields present that are not yet in the metadata profile. But by making a thorough template, we can populate it with whatever data we have and can expand in the future - does that jive with how you envisioned this?